crates/noirc_frontend/src/hir/def_collector/dc_mod.rs

```diff
@@ -49,7 +49,7 @@ pub fn collect_defs<'a>(
         });
     }
 
-    collector.collect_global_constants(context, ast.global_constants, errors);
+    collector.collect_global_constants(context, ast.globals, errors);
```

Comment:
Should we rename some of the collect_global_constants functions as well to stay consistent if we are doing the move from `global_consts` to `globals`?

Reply:
Done